index.js

```diff
@@ -6,20 +6,25 @@ const cors = require('cors')
 const app = express()
 app.use(cors())
 
-const cache = {}
+const cache = new Map();
```

Comment:
what advantage does adding a Map have over an object?
(If there are performance benefits I'm ignorant and would love to know)